authbridge/demos/github-issue/aiac/keycloak_ops/export_config.py

```
@@ -0,0 +1,362 @@
+    # Build a mapping of role/scope names to client names
+    scope_to_client = {}
+    for client in clients:
+        client_name = client["clientId"]
+        try:
+            roles = admin.get_client_roles(client["id"])
+            for role in roles:
+                role_name = role["name"]
+                # Scope names match role names (no -audience suffix)
+                scope_to_client[role_name] = client_name
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)